docs/source/process.mdx

```diff
@@ -0,0 +1,627 @@
+# Process
+
+🤗 Datasets provides many tools for modifying the structure and content of a dataset. You can rearrange the order of rows or extract nested fields into their own columns. For more powerful processing applications, you can even alter the contents of a dataset by applying a function to the entire dataset to generate new rows and columns. These processing methods provide a lot of control and flexibility to mold your dataset into the desired shape and size with the appropriate features.
+
+This guide will show you how to:
+
+- Reorder rows and split the dataset.
+- Rename and remove columns, and other common column operations.
+- Apply processing functions to each example in a dataset.
+- Concatenate datasets.
+- Apply a custom formatting transform.
+- Save and export processed datasets.
+
+Load the MRPC dataset from the GLUE benchmark to follow along with our examples:
+
+```py
+>>> from datasets import load_dataset
+>>> dataset = load_dataset('glue', 'mrpc', split='train')
+```
+
+<Tip warning={true}>
+
+All the processing methods in this guide return a new [`datasets.Dataset`]. Modification is not done in-place. Be careful about overriding your previous dataset!
+
+</Tip>
+
+## Sort, shuffle, select, split, and shard
```

Comment:
For some reason these section titles are not rendered correctly in the right bar:
![image](https://user-images.githubusercontent.com/42851186/154737960-9460c04d-1a9e-4e42-8a8c-f0b878468a93.png)

Reply:
https://github.com/huggingface/datasets/pull/3690/commits/eb7fecb0cf9ba61356758bc7316ff1bb150d8f59 Following the design choice we took on the transformers doc where we remove `<code>` format from the headings.
cc: @gary149 
<img width="850" alt="Screenshot 2022-02-21 at 09 15 55" src="https://user-images.githubusercontent.com/11827707/154968106-9c24b0e5-0149-4c89-a5b9-5a9c0db96d0f.png">